README.md

```diff
@@ -0,0 +1,190 @@
+
+---
+
+[![Discord](https://img.shields.io/discord/937092945713172480?label=discord&style=flat-square)](https://discord.gg/mkZcevnxuf)
+[![Build Status](https://github.com/open-runtimes/executor/actions/workflows/tests.yml/badge.svg)](https://github.com/open-runtimes/executor/actions/workflows/tests.yml)
+[![Twitter Account](https://img.shields.io/twitter/follow/appwrite?color=00acee&label=twitter&style=flat-square)](https://twitter.com/appwrite)
+[![Docker Pulls](https://img.shields.io/docker/pulls/openruntimes/executor?color=f02e65&style=flat-square)](https://hub.docker.com/r/openruntimes/executor)
+
+Executor for [Open Runtimes](https://github.com/open-runtimes/open-runtimes), a runtime environments for serverless cloud computing for multiple coding languages.
+
+Executor is responsible for providing HTTP API for building, starting and executing Open Runtimes. Executor is stateless and can be scaled horizontally when a load balancer is introduced in front of it. You could use any load balancer but we highly recommend using [Open Runtimes Proxy](https://github.com/open-runtimes/proxy) for it's ease of setup with Open Runtimes Executor.
+
+## Features
+
+* **Flexibility** - Configuring custom image lets you use **any** runtime for your functions.
+* **Performance** - Coroutine-style HTTP servers allows asynchronous operations without blocking. We. Run. Fast! ⚡
+* **Open Source** - Released under the MIT license, free to use and extend.
+
+## Getting Started
+
+1. Pull Open Runtimes Executor image:
+
+```bash
+docker pull openruntimes/executor
+```
+
+2. Create `docker-compose.yml` file:
+
+```yml
+version: '3'
+services:
+  openruntimes-executor:
+    container_name: openruntimes-executor
+    hostname: exc1
+    stop_signal: SIGINT
+    image: openruntimes/executor
+    networks:
+      runtimes:
+    ports:
+      - 9900:80
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - openruntimes-builds:/storage/builds:rw
+      - openruntimes-functions:/storage/functions:rw
+      - /tmp:/tmp:rw
+      - ./functions:/storage/functions:rw
+    environment:
+      - OPEN_RUNTIMES_EXECUTOR_ENV
+      - OPEN_RUNTIMES_EXECUTOR_STORAGE_DEVICE
+      - OPEN_RUNTIMES_EXECUTOR_STORAGE_S3_ACCESS_KEY=
+      - OPEN_RUNTIMES_EXECUTOR_STORAGE_S3_SECRET=
+      - OPEN_RUNTIMES_EXECUTOR_STORAGE_S3_REGION
+      - OPEN_RUNTIMES_EXECUTOR_STORAGE_S3_BUCKET=
+      - OPEN_RUNTIMES_EXECUTOR_STORAGE_DO_SPACES_ACCESS_KEY=
+      - OPEN_RUNTIMES_EXECUTOR_STORAGE_DO_SPACES_SECRET=
+      - OPEN_RUNTIMES_EXECUTOR_STORAGE_DO_SPACES_REGION
+      - OPEN_RUNTIMES_EXECUTOR_STORAGE_DO_SPACES_BUCKET=
+      - OPEN_RUNTIMES_EXECUTOR_STORAGE_BACKBLAZE_ACCESS_KEY=
+      - OPEN_RUNTIMES_EXECUTOR_STORAGE_BACKBLAZE_SECRET=
+      - OPEN_RUNTIMES_EXECUTOR_STORAGE_BACKBLAZE_REGION
+      - OPEN_RUNTIMES_EXECUTOR_STORAGE_BACKBLAZE_BUCKET=
```

Comment:
I think there is a difference between having an `=` after the env variable name

Having an `=` after the env variable essentially means we are equating it to an empty string ""
Not having an `=` after the variable indicates that it will be supplied through the environment.

Since we are supplying all vars from an external env file, we should remove `=` from all the vars

Reply:
Removed all `=`